wrapper/src/main/java/software/amazon/jdbc/ds/AwsWrapperDataSource.java

```diff
@@ -259,6 +260,49 @@ ConnectionWrapper createConnectionWrapper(
         telemetryFactory);
   }
 
+  private java.sql.Driver getTargetDriver(
```

Comment:
this function is the same than the one from Driver - is there a way to have it only once and both places use it?

Reply:
I extracted this logic to TargetDriverHelper class.